glide.lock

```diff
@@ -420,4 +419,49 @@ imports:
   - 1.5/tools/metrics
   - 1.5/tools/record
   - 1.5/transport
+- name: k8s.io/kubernetes
```

Comment:
use client-go instead? we do not want to introduce the entire k8s as dependency.